app-ingest/src/main/scala/com/azavea/rf/JobDefinition.scala

```diff
@@ -0,0 +1,55 @@
+package com.azavea.rf
+
+import spray.json._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+
+import geotrellis.vector.Extent
+import geotrellis.vector.io._
+
+import java.net.URL
+
+/** Each TileSource, which sits the bottom of any JobDefinition (see below) is made of tiff urls and a target extent */
+case class TileSource(urls: Array[String], extent: Extent)
```

Comment:
It seems to me like band information should be available inside the tiff metadata. I'll add zoom levels though, pending further suggestions from @lossyrob and @echeipesh